lib/index.js

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import Turnstile from '../components/Turnstile.vue';
+
+export {Turnstile};
```

Comment:
I'm pretty sure this can be shortened to:

```suggestion
export {default as Turnstile} from '../components/Turnstile.vue';
```